### ISSUE TITLE
Wrong file and directory permissions for gem?

### ISSUE THREAD

**Issue by CyberDeck** [tomichj/autocomplete_rails#1]
After a clean install for Rails 5.1, I cannot use the `autocomplete` method within my custom controllers. Neither with `bin/rails c` I am able to find any traces of the `AutocompleteRails::Controller`:

```ruby
[1] pry(main)> require 'autocomplete_rails'
LoadError: cannot load such file -- autocomplete_rails
from /var/lib/gems/2.3.0/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:292:in `require'
[2] pry(main)> ls AutocompleteRails::Controller
NameError: uninitialized constant AutocompleteRails
```

However, after changing the gem files for all users to be readable with 
```bash
# sudo chmod o+r -R -- /var/lib/gems/2.3.0/gems/autocomplete_rails-0.3.0
```
I can load it successfully within the console:

```ruby 
[1] pry(main)> require 'autocomplete_rails'
=> true
[2] pry(main)> ls AutocompleteRails::Controller
constants: ClassMethods
AutocompleteRails::Controller.methods: included
```

To be able to use it within my own controllers, I need to require it with an initializer in my config directory, e.g. creating a file `config/initializers/autocomplete.rb`:

```ruby 
require 'autocomplete_rails'
```

Now, I am able to use the  `autocomplete` method within my controller.

Is the current gem broken or did I miss something in regard to the installation instructions?

**Comment by tomichj** [comment]
Thanks for the very comprehensive bug report! I'll check it out and get back to you in 48 hours or less. I probably ust had my umask set too aggressively and didn't notice. Thanks again!

**Comment by tomichj** [comment]
You were correct, the permissions were incorrect. Fixed in 0.3.1. Thanks!
